to_backend_theme/static/src/scss/style.scss

```diff
@@ -441,3 +441,10 @@ textarea, select, .o_form_view.o_form_editable .o_form_field_many2manytags,
 .o_thread_window .o_thread_window_header {
 	background-color: $brand-primary;
 }
+
+.o_modal_fullscreen .o_viewer_content {
+	padding-top: 46px;
+	.o_viewer-header {
+		top: 46px;
+	}
+}
```

Comment:
Cần test đối với các vị trí chatter ở bên cạnh cũng như bên dưới. Đồng thời cũng cần test các chỗ khác mà có sử dụng o_modal_fullscreen sẽ không bị ảnh hưởng. Anh thấy trong module web_responsive có đoạn này, chắc có liên quan?
```scss
.o_modal_fullscreen .o_viewer_content {
    .o_viewer-header {
        .o_image_caption {
            display: contents;
        }

        // Now uses a container to have more buttons
        .o_buttons {
            min-width: 35px;
            text-align: right;

            // Now close button ('X') it's a fa-icon
            > .o_close_btn {
                top: unset;
                left: unset;
                bottom: unset;
                right: unset;
                font-size: unset;
                font-weight: unset;
            }
        }
    }
}
```

Reply:
@davidtranhp em test thấy ok rồi đấy anh ạ

Reply:
Thank you!